test-utils/src/test/java/org/neo4j/gds/core/CypherExporterTest.java

```diff
@@ -57,14 +59,14 @@ void testDumpByGraphDbApi() {
 
         //language=Cypher
         String expected =
-                "CREATE (n0:Label1 {foo:bar})\n" +
-                "CREATE (n1:Label1 {property:42.1337})\n" +
-                "CREATE (n2:Label2)\n" +
-                "CREATE (n3)\n" +
-                "CREATE\n" +
-                "  (n0)-[:TYPE1 {property:1337.42}]->(n2),\n" +
-                "  (n0)-[:TYPE1 {bar:baz}]->(n1),\n" +
-                "  (n1)-[:TYPE2]->(n2),\n" +
+                "CREATE (n0:Label1 {foo:bar})" + newLine +
+                "CREATE (n1:Label1 {property:42.1337})" + newLine +
+                "CREATE (n2:Label2)" + newLine +
+                "CREATE (n3)" + newLine +
+                "CREATE" + newLine +
+                "  (n0)-[:TYPE1 {property:1337.42}]->(n2)," + newLine +
+                "  (n0)-[:TYPE1 {bar:baz}]->(n1)," + newLine +
+                "  (n1)-[:TYPE2]->(n2)," + newLine +
```

Comment:
No this is not the same; this is an expected output string.
However, assertJ comes to the rescue:

```
assertThat(output.toString().trim()).isEqualToIgnoringWhitespace(expected);
```

or even

```
assertThat(output.toString().trim()).isEqualToNormalizingNewlines(expected);
```

But in this case I would go with your solution because it's probably desirable that the CypherExporter does include appropriate newlines for readability.